addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -155,7 +155,8 @@ def _generate_efaktur_invoice(self, delimiter):
             else:
                 number_ref = str(move.name) + " " + nik
 
-            street = ', '.join([x for x in (move.partner_id.street, move.partner_id.street2) if x])
+            shipping_partner = self.env['res.partner'].browse(self._get_invoice_delivery_partner_id())
```

Comment:
@agr-odoo I am afraid this should be move instead of self...  It will fail with the ensure_one.  So please do not fw-port yet.